lib/resources/r4/charge_item_modify.yaml

```diff
@@ -0,0 +1,417 @@
+    description: The current state of the charge.
+    action: modify
+    example: |
+      {
+        "status": "unknown"
+      }
+    note: >
+      <ul>
+        <li>The status must be set to `unknown`</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.status
+
+  - name: subject
+    required: 'Yes'
+    type: Reference
+    description: The patient associated with the charge.
+    action: modify
+    example: |
+      {
+        "subject": {
+          "reference": "Patient/12768550"
+        }
+      }
+    note: >
+      <ul>
+        <li>Must be a Patient reference</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.subject
+
+  - name: context
+    required: 'Yes'
+    type: Reference
+    description: The encounter associated with the charge.
+    action: modify
+    example: |
+      {
+        "context": {
+          "reference": "Encounter/97965088"
+        }
+      }
+    note: >
+      <ul>
+        <li>Must be an Encounter reference</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.context
+
+  - name: occurrenceDateTime
+    required: 'No'
+    type: dateTime
+    description: Date/time when the charged service was applied.
+    action: modify
+    example: |
+      {
+        "occurrenceDateTime": "2021-01-08T10:38:00Z"
+      }
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.occurrenceDateTime
+
+  - name: quantity
+    required: 'No'
+    type: Quantity
+    description: Quantity of which the charge item has been serviced.
+    action: modify
+    example: |
+      {
+        "quantity": {
+          "value": 31
+        }
+      }
+    note: >
+      <ul>
+        <li>The value must be in the form of `integer`</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.quantity
+
+  - name: code
+    required: 'Yes'
+    type: CodeableConcept
+    description: The primary billing code that identifies the charge.
+    action: modify
+    example: |
+      {
+        "code": {
+          "coding": [
+            {
+              "extension": [
+                {
+                  "valueCoding": {
+                    "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/14002",
+                    "code": "667687"
+                  },
+                  "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/bill-code-schedule"
+                }
+              ],
+              "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/CodeSystem/BillCodes-CDM_SCHED",
+              "code": "64897"
+            }
+          ]
+        }
+      }
+    note: >
+      <ul>
+        <li>The coding field must contain only one coding from the CDM_SCHED, CPT, HCPCS, or PROCCODE proprietary Bill Codes Types systems.</li>
+        <li>Tiering logic from highest to lowest: CDM_SCHED, CPT, HCPCS, or PROCCODE</li>
```

Comment:
I believe it would be helpful to provide some further context for the tiering logic and how it relates to this field. How do we feel about `The Bill Code Type provided in `code` must be the highest tier of all of the Bill Code Types associated to the ChargeItem`?

cc @alexbeckner1

Reply:
I like this additional note. Should it go right before the note about the Tiering logic?

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/780/commits/4595890bdc6f918a94d93cd54ca164dab864e8e0#diff-3c565d9c41e2bb3d417efe63a774683e72b07cd38f2ff54a695c695999f29938R127

Reply:
I like Alex's suggestion of putting it before the tiering note.

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/780/commits/8854164a3591fb59fce490a6fae6aa0e29c11429#diff-3c565d9c41e2bb3d417efe63a774683e72b07cd38f2ff54a695c695999f29938R145